doc/api/errors.md

```diff
@@ -2821,25 +2821,6 @@ An unspecified or non-specific system error has occurred within the Node.js
 process. The error object will have an `err.info` object property with
 additional details.
 
-<a id="ERR_TAP_LEXER_ERROR"></a>
```

Comment:
There is a section on this page for legacy errors that are no longer used. I think these should be moved there?

Reply:
Moved it under the correct section!